core/templates/dev/head/pages/exploration_player/PlaythroughService.js

```diff
@@ -182,14 +189,22 @@ oppia.factory('PlaythroughService', [
 
     return {
       initSession: function(newExplorationId, newExplorationVersion) {
+        isPlayerInSamplePopulation =
+          this.determineIfPlayerIsInSamplePopulation();
         playthrough = PlaythroughObjectFactory.createNew(
           null, newExplorationId, newExplorationVersion, null, {}, []);
         expStopwatch = StopwatchObjectFactory.create();
       },
+      determineIfPlayerIsInSamplePopulation: function() {
```

Comment:
Why is this public? Shouldn't this also be a private function?

Reply:
@brianrodri If this becomes a private function, we wouldn't be able to spy on it and change its return value in tests, would we?

Reply:
Hmm, it'd make more sense to swap these b/c this function is what actually performs the "implementation details". The other is a simple boolean check, which is much easier to mock.

So:
`_determineIfPlayerIsInSamplePopulation` would be the implementation details
`isPlayerExcludedFromSamplePopulation` would be the function you can mock (and allow the rest of your internals to use).